server/routes.go

```diff
@@ -35,9 +35,9 @@ func addRoutes(mux *http.ServeMux, sessions *session.Sessions, strg *storage.Sto
 	// Dynamic content
 	mux.Handle("/", page.MainPageHandler())
 	mux.Handle("GET /active", page.ActiveHandler(sessions, strg))
-	mux.Handle("GET /u/", http.StripPrefix("/u", page.UserContent(sessions, strg)))
+	mux.Handle("/u/", http.StripPrefix("/u", page.UserContentHandler(sessions, strg)))
```

Comment:
Optional: you can try to separate it like this, but only if you think it makes the code easier to read:
```go
// Serve user content
mux.Handle("GET /u/", http.StripPrefix("/u", page.UserContentGet(sessions, strg)))
// Post new user content
mux.Handle("POST /u/", http.StripPrefix("/u", page.UserContentPost(sessions, strg)))
```